<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Listen</string>
      </property>
     </widget>
     <widget class="QLabel" name="serverMessageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>501</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QLineEdit" name="fileNameLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose</string>
      </property>
     </widget>
     <widget class="QLabel" name="clientMessageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>391</width>
        <height>401</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="mediumButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>500</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Medium</string>
      </property>
     </widget>
     <widget class="QPushButton" name="averageButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Average</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>460</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QPushButton" name="maxButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onListen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onChoose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>averageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAverage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mediumButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMedium()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onListen()</slot>
  <slot>onMin()</slot>
  <slot>onMax()</slot>
  <slot>onAverage()</slot>
  <slot>onMedium()</slot>
  <slot>onChoose()</slot>
 </slots>
</ui>
